source/agora/common/Hash.d

```diff
@@ -157,6 +157,12 @@ public void hashPart (scope cstring record, scope HashDg state) /*pure*/ nothrow
     state(cast(const ubyte[])record);
 }
 
+/// Ditto
+public void hashPart (scope const(ubyte)[] record, scope HashDg state) /*pure*/ nothrow @nogc @trusted
```

Comment:
Since that's just calling `state`, I don't think this overload is really useful ?

Reply:
Ah yeah that was stupid of me.